pkg/backup/compaction_job.go

```diff
@@ -263,7 +264,7 @@ func (b *backupResumer) ResumeCompaction(
 			return err
 		}
 
-		if err := execCtx.ExecCfg().JobRegistry.CheckPausepoint("backup.before.write_first_checkpoint"); err != nil {
+		if err := execCtx.ExecCfg().JobRegistry.CheckPausepoint("backup_compaction.before.write_first_checkpoint"); err != nil {
```

Comment:
are these `before/after.write_first_checkpoint` pausepoints even used in tests?

Reply:
Nope they are not --- originally I just piggy-backed off of the original `backup.*` pausepoints, but
I realized that perhaps we don't want to have backup pausepoints affecting compaction since they are
fundamentally two different processes. We could remove them of course, but I was under the
impression that pausepoints are used in our tests as well as our own live tests when we are
debugging.

Reply:
i'd remove them. they really are only ever used in tests.